src/components/common/navigation/ScrollToTop.js

```diff
@@ -0,0 +1,16 @@
+import { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+
+class ScrollToTop extends Component {

```

Comment:
can u convert it to function component? We'll try to stay away from class components

Reply:
I have spent about 30 minutes trying to convert it to a functional component and I really do not think it is worth it. The main problem is that I cannot access prevProps with useEffect hook, I would need to create a custom hook and it looks like shit IMO, not worth the trouble:
https://stackoverflow.com/questions/53446020/how-to-compare-oldvalues-and-newvalues-on-react-hooks-useeffect

Reply:
but you don't have to specifically create a custom hook for it... You could just use part of the code that has been supplied on stack 
```
useEffect(() => {
      window.scrollTo(0, 0)
    }, [location])
```
it's because when you add this second argument `[location]` this useEffect should execute only when the value of this field (location) has been changed, so you don't really need prevProps I guess..